convert-to-json.py

```diff
@@ -15,22 +15,23 @@
 
 system_name = os.path.basename(os.path.realpath(folder_name))
 
-with open(os.path.join(folder_name, "scf.in")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.in")) as fhandle:
```

Comment:
I think here we should keep the convention that's defined above, i.e. forcing to use filenames of `scf.in`, `scf.out`, `matdyn.modes`. Would it make sense to rename the files as such? then in principle it's easy to regenerate the json files without having to manually change this code here.

Reply:
I agree, I was not considering this issue, I'll change it back (and modifiy the filenames accordingly)